proposal for how the team operates (#215)
rust-lang/rust-project-goals: src/admin/cfp.md

# Call for proposals

> Each goal milestone corresponds to six months, designated in the format YYYYHN, e.g., 2024H2 or 2025H1. To launch a new goal season, you should get started a month or two before the new season starts:

Reviewer: ```suggestion
Each goal milestone corresponds to six months, designated in the format YYYYhN, e.g., 2024h2 or 2025h1. To launch a new goal season, you should get started a month or two before the new season starts:
```

Author: We are I think a bit inconsistent about this.

Author: The directions are 2024h1 but other places we write 2024H1, for example.

> * For an H1 season, start around mid Oct or November of the year before.
> * For an H2 season, start around mid April or May of the year before.

Reviewer: ```suggestion
* For an H1 season, start around mid October of the year before.
* For an H2 season, start around mid April of the year before.
```
